features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -440,11 +440,16 @@
     "/api/identity/config-mgt/v1.0/",
     "/api/users/v2/me/webauthn",
     "/api/",
-    "/user-portal/"
+    "/user-portal/",
+    "/authenticationendpoint/",
+    "/accountrecoveryendpoint/"
   ],
   "tenant_context.rewrite.servlets": [
     "/identity/(.*)",
-    "(.*)/identity/metadata/saml2"
+    "(.*)/identity/metadata/saml2",
```

Comment:
is .* needed here. IIRC tenant context valve append the respective tenant patterns that needs to be verified